lib/versions.js

```diff
@@ -60,3 +60,15 @@ function fetchLatestTag(repo) {
     },
   );
 }
+
+function getVersionFromToolVersionsFile() {
+  const fileName = ".tool-versions";
+  if (!fs.existsSync(fileName)) return;
+
+  const toolVersions = fs.readFileSync(fileName, "utf-8").split("\n");
+  const scarbVersionInfo = toolVersions.find((tool) => tool.match(/scarb/));
```

Comment:
Make sure this only matches lines that start with `scarb` and ignores comments.

e.g. https://github.com/asdf-vm/actions/blob/8b8467c5522bb0aea4977082037e4f2956d67e52/src/plugins-add/index.ts#L50C23-L50C23

Reply:
Good catch, thanks!